pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -43,6 +45,13 @@ func (i *BootstrapImage) Generate(p asset.Parents) error {
 	defer cancel()
 	switch config.Platform.Name() {
 	case baremetal.Name:
+		// Check for RHCOS image URL override
+		if osimage = config.Platform.BareMetal.BootstrapMachineOSImage; osimage != "" {
+			logrus.Infof("Found an URL override in Baremetal Platform for Bootstrap Machine OS Image: %s", osimage)
+			*i = BootstrapImage(osimage)
+			return nil
```

Comment:
the switch only sets the appropiate osimage and err variables.. and the rest is handled already outside the switch

Reply:
Ack. Will change.

Reply:
Changes committed.